src/components/Comments/Comment.less

```diff
@@ -148,6 +149,14 @@
     }
   }
 
+  &__replies--no-indent {
```

Comment:
Wouldnt be simpler to do handle the intend just with css like this `&__replies > &__replies > &__replies` ?

Reply:
It would match only 3rd level replies. However, there can be 4th, 5th ... nth level. Not really viable to handle this with selectors.